packages/process/process.pony

```diff
@@ -21,8 +21,7 @@ use "files"
 actor Main
   new create(env: Env) =>
     // create a notifier
-    let client = ProcessClient(env)
-    let notifier: ProcessNotify iso = consume client
+    let client: ProcessClient iso = ProcessClient(env)
```

Comment:
well, why bother with the `: ProcessClient iso`?

that can go as well.

Reply:
I kept it for two reasons:

1. The original did it.
2. If someone is reading the example then they have to look in one of three places to realize that ProcessMonitor takes a ProcessNotify iso.
  - ProcessClient.create()
  - The ProcessMonitor.create function signature, or
  - The ProcessMonitor callsite in the example (and infer that the consume likely means it was originally an iso).

For somebody unfamiliar with the package - I wanted that to be explicit.

If you'd still rather lose the definition I can lose it, but that is why I chose to be explicit in this case.